test/extensions/filters/network/redis_proxy/conn_pool_impl_test.cc

```diff
@@ -1032,11 +1032,13 @@ TEST_F(RedisConnPoolImplTest, HostsAddedAndEndWithClusterRemoval) {
 
 TEST_F(RedisConnPoolImplTest, MakeRequestToRedisCluster) {
 
-  absl::optional<envoy::config::cluster::v3::Cluster::CustomClusterType> cluster_type;
-  cluster_type.emplace();
-  cluster_type->set_name("envoy.clusters.redis");
+  envoy::config::cluster::v3::Cluster::CustomClusterType cluster_type_;
+  cluster_type_.set_name("envoy.clusters.redis");
+
+  OptRef<const envoy::config::cluster::v3::Cluster::CustomClusterType> cluster_type =
+      makeOptRef<const envoy::config::cluster::v3::Cluster::CustomClusterType>(cluster_type_);
   EXPECT_CALL(*cm_.thread_local_cluster_.cluster_.info_, clusterType())
-      .WillOnce(ReturnRef(cluster_type));
+      .WillOnce(Return(cluster_type));
```

Comment:
nit: Can this directly return `makeOptRef(cluster_type_)` (avoiding the temp var above)?

Reply:
Done